index.bs

```diff
@@ -446,32 +436,120 @@ partial interface NeuralNetworkContext {
 
     **Returns:** an {{Operand}}. The output tensor that contains the result of
     element-wise binary multiplication of the two input tensors.
-
-    Computes the element-wise binary multiplication of the two input tensors.
+  
     The element-wise binary multiplication will be broadcasted according to
     [[!numpy-broadcasting-rule]]. The rank of the output tensor is the maximum
     rank of the input tensors. For each dimension of the output tensor, its size
     is the maximum size along that dimension of the input tensors.
 </div>
 
+### pool2d ### {#api-neuralnetworkcontext-pool2d}
+Compute a *mean*, *L2 norm*, or *max* reduction operation across all the elements within the moving window over the input tensor. See the description of each type of reduction in [[#api-neuralnetworkcontext-reduce]].
+<script type=idl>
+partial interface NeuralNetworkContext {
+  Operand averagePool2d(Operand input, optional sequence<long> windowDimensions,
+                        optional sequence<long> padding, optional sequence<long> strides,
+                        optional sequence<long> dilations, optional OperandLayout layout = "nchw");
+  Operand l2Pool2d(Operand input, optional sequence<long> windowDimensions,
+                  optional sequence<long> padding, optional sequence<long> strides,
+                  optional sequence<long> dilations, optional OperandLayout layout = "nchw");
+  Operand maxPool2d(Operand input, optional sequence<long> windowDimensions,
+                    optional sequence<long> padding, optional sequence<long> strides,
+                    optional sequence<long> dilations, optional OperandLayout layout = "nchw");
+};
+</script>
+<div algorithm=pool2d>
+    **Arguments:**
+        - *input*: an {{Operand}}. The input 4-D tensor. The logical shape
+            is interpreted according to the value of *layout*.
+        - *windowDimensions*: an optional sequence of {{long}} of length 2. The dimensions of the sliding window,
+            [window_height, window_width]. If not present, the window dimensions are assumed to be the last 
+            two dimensions of the input shape.
```

Comment:
For "nhwc" layout, the `height` and `width` are not the last two dimensions of the input shape.

Reply:
Good catch. Fixed. @huningxin

Reply:
looks good, thanks!